packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -756,6 +813,12 @@ class AccountInternal extends PureComponent {
       const idSet = new Set(ids);
 
       transactions.forEach(trans => {
+        if (name === 'cleared' && trans.reconciled === true) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
        if (name === 'cleared' && trans.reconciled) {
```